contrib/net/http/roundtripper_test.go

```diff
@@ -34,6 +35,8 @@ func TestRoundTripper(t *testing.T) {
 	}))
 	defer s.Close()
 
+	listenIP, listenPort, _ := net.SplitHostPort(s.Listener.Addr().String())
```

Comment:
```suggestion
	ip, port, _ := net.SplitHostPort(s.Listener.Addr().String())
```
Let's keep everything short.